lib/cdc/util/geometry_test.go

```diff
@@ -0,0 +1,17 @@
+package util
+
+import "github.com/stretchr/testify/assert"
+
+func (u *UtilTestSuite) TestParseGeometryPoint() {
+	{
+		geoJSONString, err := parseGeometryPoint(map[string]interface{}{
+			"x":    2.2945,
+			"y":    48.8584,
+			"wkb":  "AQEAAABCYOXQIlsCQHZxGw3gbUhA",
```

Comment:
Do we need to do anything with `wkb` or `srid`?

Reply:
Not yet, I left it out for now - I'll leave a comment.